jvm/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat.*
+import org.gradle.api.tasks.testing.logging.TestLogEvent.*
+
+plugins {
+    id("org.jetbrains.kotlin.jvm")
+    id("java-library")
+    id("maven-publish")
+    id("signing")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+    withSourcesJar()
+}
+
+tasks.withType<Test> {
+    useJUnitPlatform()
+
+    testLogging {
+        events(PASSED, SKIPPED, FAILED, STANDARD_OUT, STANDARD_ERROR)
+        exceptionFormat = FULL
+        showExceptions = true
+        showCauses = true
+        showStackTraces = true
+    }
+}
+
+dependencies {
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7")
+    implementation("net.java.dev.jna:jna:5.8.0")
+    api("org.slf4j:slf4j-api:1.7.30")
+    testImplementation("junit:junit:4.13.2")
+    testRuntimeOnly("org.junit.vintage:junit-vintage-engine:5.8.2")
```

Comment:
For some reason I had to add this dependency for the `./gradlew :jvm:test` command to work.

Reply:
Same with the migration to the new API! This PR was initially made against the older version of bdk-ffi and so it was working, but when I worked on updating the tests I also found I needed to add this `junit-vintage-engine`. Thanks for fixing!